backend/functions/lib/testing/mock-data/case-history.mock.ts

```diff
@@ -12,6 +12,7 @@ export const CASE_HISTORY: CaseAssignmentHistory[] = [
         id: '2265215f-88e8-4cc7-a2d1-2d5ebaaabe73',
         documentType: 'ASSIGNMENT',
         caseId: '081-22-84687',
+        userId: 'userId-Susan Arbeit',
```

Comment:
We have a bunch of tests where we redefine a set of mock attorney users. Maybe we should just define a mock set once and reuse them in our many tests.

Reply:
This will beleft alone because this is Case Assignment history rather than directly on Attorney's